x-pack/platform/test/alerting_api_integration/common/lib/actions_simulations_utils.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  getExternalServiceSimulatorPath,
+  getAllExternalServiceSimulatorPaths,
+  ExternalServiceSimulator,
+} from '@kbn/actions-simulators-plugin/server/plugin';
+import { getServiceNowServer } from '@kbn/actions-simulators-plugin/server/plugin';
+import { RecordingServiceNowSimulator } from '@kbn/actions-simulators-plugin/server/servicenow_simulation';
+import {
+  tinesAgentWebhook,
+  tinesStory1,
+} from '@kbn/actions-simulators-plugin/server/tines_simulation';
+
+export interface ServiceNowRequest {
+  work_notes?: string;
+}
+
+export {
+  getExternalServiceSimulatorPath,
+  getAllExternalServiceSimulatorPaths,
+  ExternalServiceSimulator,
+  getServiceNowServer,
+  RecordingServiceNowSimulator,
+  tinesAgentWebhook,
+  tinesStory1,
+};
```

Comment:
This is a workaround I had to do because direct plugin imports in `x-pack/test/` started to throw TS errors after plugin relocation to `x-pack/platform/test/`:

Fails: 
```
import {..} from '@kbn/actions-simulators-plugin/server/plugin';
```

Works:
```
import {..} from '@kbn/test-suites-xpack-platform/alerting_api_integration/common/lib/actions_simulations_utils';
```